src/model_signing/manifest.py

```diff
@@ -263,3 +404,24 @@ def resource_descriptors(self) -> Iterator[ResourceDescriptor]:
         """Yields each resource from the manifest, one by one."""
         for item, digest in sorted(self._item_to_digest.items()):
             yield ResourceDescriptor(identifier=str(item), digest=digest)
+
+    @property
+    def model_name(self) -> str:
+        """The name of the model when serialized.
+
+        This is the final component of the path and is only informative.
```

Comment:
should the final component of the path be part of the docstring for the initializer arg?

Reply:
Added